cli/js/net_test.ts

```diff
@@ -54,7 +54,7 @@ testPerm({ net: true }, async function netTcpConcurrentAccept(): Promise<void> {
   const p1 = listener.accept().catch(checkErr);
   await Promise.race([p, p1]);
   listener.close();
-  await [p, p1];
+  await Promise.all([p, p1]);
   assertEquals(acceptErrCount, 1);
 });
```

Comment:
Very nice that you’re catching errors like this! 
(Sad that neither typescript nor eslint caught this.)